Src/LinearSolvers/MLMG/AMReX_MLLinOp.H

```diff
@@ -178,7 +178,8 @@ public:
     bool getEnforceSingularSolvable () const noexcept { return enforceSingularSolvable; }
 
     virtual BottomSolver getDefaultBottomSolver () const { return BottomSolver::bicgstab; }
-    virtual int getNComp () const { return 1; }
+    virtual void setNComp (int ncomp) noexcept { m_ncomp = ncomp; }
+    virtual int getNComp () const { return m_ncomp; }
```

Comment:
There is no need to add `int m_ncomp` because this is a virtual function.  This allows MLEBABecLap to override without changing MLLinOp.

Reply:
thank you, Ill revert these changes and modify only MLEBABecLap